one_fm/operations/doctype/employee_schedule/employee_schedule.py

```diff
@@ -34,8 +34,16 @@ def on_update(self):
 			)
 			if employee_schedule:
 				frappe.db.set_value("Employee Schedule", employee_schedule.name, "day_off_ot", 0)
-
+    
+	def validate_schedule_date(self):
+		"""Validate that the schedule date is not in the past"""
+		today = frappe.utils.getdate()
```

Comment:
Line 9 we have getdate imported use the same

Reply:
updated @pjamsheer